content/sensu-core/2.0/reference/entities.md

```diff
@@ -0,0 +1,85 @@
+---
+title: "Entities"
+description: "The entities reference guide."
+weight: 1
+version: "2.0"
+product: "Sensu Core"
+platformContent: true
+menu:
+  sensu-core-2.0:
+    parent: reference
+---
+
+# Entities
+
+### Why "Entities"?
+
+Sensu 2.0 uses an updated data model to address the latest paradigm shift in infrastructure and software delivery like containers and "serverless". Sensu is no longer confined to the concept of a client, we have "entities" that can can represent anything.
+
+### What is a Sensu entity?
+
+Entities, formally known as clients in Sensu 1.x, are monitoring agents (e.g. server, container, etc) that need to be monitored. They contain system information such as the hostname, OS, platform, and version. Entities not only provide context to event data but an event's uniqueness is determined by the check name and the ID of the entity upon which the check ran.
+
+## Agent entities
+
+### What is an agent entity?
+
+An agent entity is a Sensu entity which falls under the "agent" class. Agent entities are monitoring agents, which are installed and run on every system that needs to be monitored. The entity is responsible for registering the system with Sensu, sending keepalive messages (the Sensu heartbeat mechanism), and executing monitoring checks. Each entity is a member of one or more subscriptions – a list of roles and/or responsibilities assigned to the system (e.g. a webserver, database, etc). Sensu entities will "subscribe" to (or watch for) check requests published by the Sensu server (via the Sensu Transport), execute the corresponding requests locally, and publish the results of the check back to the transport (to be processed by a Sensu server).
```

Comment:
First sentence ("An agent entity is a Sensu entity which falls under the "agent" class") is programming info that doesn't make it clearer to a user. I think it can be removed unless you want to reword to in a helpful way to the user.

Reply:
`Class` is actually a field of an Entity that is visible to the user, which describes whether it is an "agent" entity or "proxy" entity. But let me reword that so the point is clear, I totally see where you're coming from with the use of "class."

Reply:
FWIW, users can specify classes of entities that are not "agent" or "proxy." Those are just reserved words for us to use internally.

Reply:
So it sounds like the contents of that field are trivial in this section then ;)